### PR TITLE
Dijsktra Algorithm

### LINKED ISSUE #5340: Dijsktra Algorithm
Dijsktra Algorithm is a greedy algorithm which is the shortest path algorithm.I would like to add it in the JAVA section.
Please assign it to me.

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
Fixes #5340 
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

(Write your answer here.)

### Description

I have added a greedy algorithm in the JAVA section and the algorithm is Dijsktra Algorithm which is used to find the shortest path in a graph from the source vertex.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
This Pr solves the issue #5340

### CODE REVIEW

**Comment by Khushi-agarwal** [comment]
@bhav09 @HarshCasper Kindly review it.

File: Java/Greedy_Algorithms/Dijsktra.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  
+        return min_index;
+    }
+  
+    // A utility function to print the constructed distance array
+    void printSolution(int dist[])
+    {
+        System.out.println("Vertex \t & Distance from Source");
+        for (int i = 0; i < Ver; i++)
+            System.out.println(i + " \t\t " + dist[i]); //Prints the distance in the form of table
```
Comment (ankitaggarwal23) [inline Java/Greedy_Algorithms/Dijsktra.java:30]: Remove redundant comments

File: Java/Greedy_Algorithms/Dijsktra.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        Scanner sc=new Scanner(System.in);
+        System.out.println("Enter number of vertices");
+        Ver=sc.nextInt();
+        int graph[][] = new int[Ver][Ver];
+        System.out.println("Enter graph");
+        for(int i=0;i<Ver;i++)
+        {
+            for(int j=0;j<Ver;j++)
+            {
+                graph[i][j]=sc.nextInt();//creation of graph
```
Comment (ankitaggarwal23) [inline Java/Greedy_Algorithms/Dijsktra.java:72]: Remove redundant comments

File: Java/Greedy_Algorithms/Dijsktra.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
+/*Problem Description-
+This is a shortest path algorithm used in graphs which is used to find the shortest distance from a starting vertex to all the vertex.
+The edge between two vertices having minimum distance is considered and also no vertex is left alone and not connected*/
```
Comment (ankitaggarwal23) [inline Java/Greedy_Algorithms/Dijsktra.java:3]: rewrite it as,

```
/*

Problem Description-
This is a shortest path algorithm used in graphs which is used to find 
the shortest distance from a starting vertex to all the vertex. The edge 
between two vertices having minimum distance is considered and also no vertex 
is left alone and not connected.

*/
```

File: Java/Greedy_Algorithms/Dijsktra.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        for(int i=0;i<Ver;i++)
+        {
+            for(int j=0;j<Ver;j++)
+            {
+                graph[i][j]=sc.nextInt();//creation of graph
+            }
+        } 
+        dijkstra t = new dijkstra();
+        t.dijkstra(graph, 0);
+}
```
Comment (ankitaggarwal23) [inline Java/Greedy_Algorithms/Dijsktra.java:77]: Add a tab space and align braces

File: Java/Greedy_Algorithms/Dijsktra.java [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+                min_index = v;
+            }
+  
+        return min_index;
+    }
+  
+    // A utility function to print the constructed distance array
+    void printSolution(int dist[])
+    {
+        System.out.println("Vertex \t & Distance from Source");
```
Comment (ankitaggarwal23) [inline Java/Greedy_Algorithms/Dijsktra.java:28]: remove `&`

File: Java/Greedy_Algorithms/Dijsktra.java [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+6
+7
+0
+Vertex 	 & Distance from Source
+0 		 0
+1 		 9
+2 		 2
+*/
+/*Explanation-Since the edges distance has been entered thus it calculates the shortest path distance form the source vertex which
+is being taken as 0,that is, the first vertex*/
```
Comment (ankitaggarwal23) [inline Java/Greedy_Algorithms/Dijsktra.java:99]: rewrite it as,
```
/*
Input/Output

Enter number of vertices
3
Enter graph
0
0
2
0
0
0
6
7
0

Vertex 	 Distance from Source
0 		 0
1 		 9
2 		 2

Time Complexity:
Space Complexity:

*/
```
- Add Time and Space Complexity at the end of the Code   
- Remove the explanation

File: Java/README.md [hunk 7]
```diff
@@ -24,6 +24,7 @@
 - [Activity Selection](Greedy_Algorithms/Active_Selection.java)
 - [Fractional Knapsack](Greedy_Algorithms/FractionalKnapsack.java)
 - [Job_Scheduling](Greedy_Algorithms/Job_Scheduling.java)
+- [Dijsktra](Greedy_Algorithms/Dijsktra.java)
```
Comment (ankitaggarwal23) [inline Java/README.md:27]: rewrite only this part as `- [Dijsktra Algorithm]`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by Khushi-agarwal** [comment]
@ankitaggarwal23 I have made all the changes as told by you and have created a new pr which is #5914.Kindly have a look.

**Review by ankitaggarwal23 — COMMENTED** [review]
Why you are creating new PR everytime for changes requested this is the last time don't repeat do the changes in same PR.

**Comment by Khushi-agarwal** [comment]
Actually my system crashed so that's why I couldn't access this,so had to create a new pr. Sorry for the inconvenience.

**Comment by Khushi-agarwal** [comment]
> Why you are creating new PR everytime for changes requested this is the last time don't repeat do the changes in same PR.

Actually my system crashed so that's why I couldn't access this,so had to create a new pr. Sorry for the inconvenience.

**Comment by ankitaggarwal23** [comment]
Okay I already merged

**Pull request closed without merging**